tests/k8s_env/ksp/ksp_test.go

```diff
@@ -109,12 +109,13 @@ var _ = Describe("Ksp", func() {
 			// Start KubeArmor Logs
 			err = KarmorLogStart("policy", "multiubuntu", "Network", ub1)
 			Expect(err).To(BeNil())
-
-			sout, _, err := K8sExecInPod(ub1, "multiubuntu",
-				[]string{"bash", "-c", "ping -c 1 127.0.0.1"})
-			Expect(err).To(BeNil())
-			fmt.Printf("OUTPUT: %s\n", sout)
-			Expect(sout).To(MatchRegexp("PING.*127.0.0.1"))
+			Eventually(func() string {
+				sout, _, err := K8sExecInPod(ub1, "multiubuntu",
+					[]string{"bash", "-c", "ping -c 1 127.0.0.1"})
+				Expect(err).To(BeNil())
+				fmt.Printf("OUTPUT: %s\n", sout)
+				return sout
+			}, 10*time.Second, 2*time.Second).Should(MatchRegexp("PING.*127.0.0.1"))
```

Comment:
We don't need eventually for Audit Policies, Let's refrain from adding that here.